src/WebJobs.Script/Config/NameResolver.cs

```diff
@@ -2,17 +2,22 @@
 // Licensed under the MIT License. See License.txt in the project root for license information.
 
 using System;
-using System.Configuration;
 using System.Globalization;
 
 namespace Microsoft.Azure.WebJobs.Script.Config
 {
-    internal class NameResolver : INameResolver
+    internal class NameResolver : DefaultNameResolver
```

Comment:
Not sure if it matters, but this change means that any app setting of 'rand-guid' will be returned rather than an actual random guid. The change likely won't affect anyone but thought I'd call it out just in case that wasn't intentional.

---
In reply to: [70362398](https://github.com/Azure/azure-webjobs-sdk-script/pull/488#discussion_r70362398) [](ancestors = 70362398)